### PR TITLE
Update GUI for release

### PR DESCRIPTION
**Pull request by GeorgWa**
- Align default GUI and yaml parameters
- fix peptdeep checkpoint bug

### CODE REVIEW

File: alphadia/libtransform.py [hunk 1]
```diff
@@ -328,7 +328,7 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
             logging.info(f"Loading PeptDeep models of type {self.peptdeep_model_type}")
             model_mgr.load_installed_models(self.peptdeep_model_type)
 
-        if self.peptdeep_model_path is not None:
+        if self.peptdeep_model_path is not None and self.peptdeep_model_path != "":
```
Comment (mschwoer) [inline alphadia/libtransform.py:331]: `if self.peptdeep_model_path`?

**Review by mschwoer — APPROVED** [review]
LGTM

File: alphadia/libtransform.py (hunk 1 above)
Comment (mschwoer) [reply alphadia/libtransform.py:331]: should we run e2e tests here?
(classic 'submarine' PR: disguised as UI change, but changes default parameters ;-))

Comment (GeorgWa) [reply alphadia/libtransform.py]: Yes, to some degreee :D But all of these parameters are redefined in the e2e tests. So we only change to enable Oscars optimization by default.

File: alphadia/libtransform.py [hunk 2]
```diff
@@ -324,11 +324,11 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
         model_mgr = ModelManager(device=device)
 
         # will load other model than default generic
-        if self.peptdeep_model_type is not None:
+        if self.peptdeep_model_type:
             logging.info(f"Loading PeptDeep models of type {self.peptdeep_model_type}")
             model_mgr.load_installed_models(self.peptdeep_model_type)
 
-        if self.peptdeep_model_path is not None and self.peptdeep_model_path != "":
+        if self.peptdeep_model_path and self.peptdeep_model_path != "":
```
Comment (mschwoer) [inline alphadia/libtransform.py:331]: just `if self.peptdeep_model_path` will do .. `""` is falsy

**Pull request closed without merging**